pkg/minikube/constants/constants.go

```diff
@@ -39,6 +40,9 @@ const MinikubeContext = "minishift"
 // MiniShiftEnvPrefix is the prefix for the environmental variables
 const MiniShiftEnvPrefix = "MINISHIFT"
 
+// Variable for registrator
+var Registrator registration.Registrator
```

Comment:
This is something I want to try so it's easy to use this `Registrator` variable to unregister a VM without initiate it again but looks like it not working as expected.